sources/dynamodb/schema_test.go

```diff
@@ -85,6 +89,23 @@ func TestProcessSchema(t *testing.T) {
 				},
 			},
 		},
+		{
+			Table: &dynamodb.TableDescription{
+				TableName: &tableNameA,
+				KeySchema: []*dynamodb.KeySchemaElement{
+					{AttributeName: &attrNameA, KeyType: &hashKeyType},
+				},
+			},
+		},
+		{
+			Table: &dynamodb.TableDescription{
+				TableName: &tableNameB,
+				KeySchema: []*dynamodb.KeySchemaElement{
+					{AttributeName: &attrNameA, KeyType: &hashKeyType},
+					{AttributeName: &attrNameB, KeyType: &sortKeyType},
+				},
+			},
+		},
```

Comment:
Need to add additional DescribeTableOutput here as DescribeTable is called twice now. Once each for GetConstraints and GetIndexes.